### PR TITLE
lint: format js

### PR DESCRIPTION
**Pull request by nobkd**
Most changes are formatting or whitespace.

Imports are now categorized in 3 categories and ordered by alphabet respectively:

- `node:*` internals
- package imports
- internal imports / imports from the current package

After imports and top level variable definitions, there are two newlines separating them from the actual code. 
Exactly one newline at the end of the js file.

### CODE REVIEW

File: packages/nuekit/src/layout/head.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ export function renderHead(data) {
     const css = inline_css.map(el => el.css).join('')
     head.push(elem('style', css))
 
-  // dev mode: keep path info for hot-reloading
+    // dev mode: keep path info for hot-reloading
```
Comment (nobkd) [inline packages/nuekit/src/layout/head.js]:
```suggestion
  // dev mode: keep path info for hot-reloading
```

File: packages/glow/src/glow.js [hunk 2]
```diff
@@ -166,7 +166,7 @@ export function parseRow(row, lang) {
   for (const el of tags) {
     const { re, shift } = el
 
-    row.replace(re, function(match, start, n) {
+    row.replace(re, function (match, start, n) {
```
Comment (tipiirai) [inline packages/glow/src/glow.js]: I personally prefer not to have a whitespace here. Or is this some sort of standard?

File: packages/glow/src/glow.js [hunk 3]
```diff
@@ -263,7 +263,7 @@ export function parseSyntax(str, lang) {
 
 
 // code, { language: 'js', numbered: true }
-export function glow(str, opts={}) {
+export function glow(str, opts = {}) {
```
Comment (tipiirai) [inline packages/glow/src/glow.js:266]: No whitespace after the method name here. Was this a slip?

**Comment by tipiirai** [comment]
I'm good with the changes, except for the whitespace prior function name — which I'd like to discuss more

**Comment by nobkd** [comment]
This part was automatically formatted, but I also don't like the space there, so I changed it.

**Pull request merged**